mmhuman3d/data/data_converters/gta_human.py

```diff
@@ -29,30 +30,33 @@ class GTAHumanConverter(BaseConverter):
     <https://arxiv.org/pdf/2110.07588.pdf>`__.
     """
 
-    focal_length = 1158.0337
-    camera_center = (960, 540)  # xy
-    image_size = (1080, 1920)  # (height, width)
-
-    device = torch.device(
-        'cuda') if torch.cuda.is_available() else torch.device('cpu')
-
-    smpl = build_body_model(
-        dict(
-            type='SMPL',
-            keypoint_src='smpl_54',
-            keypoint_dst='smpl_49',
-            model_path='data/body_models/smpl',
-            extra_joints_regressor='data/body_models/J_regressor_extra.npy')
-    ).to(device)
-
-    camera = build_cameras(
-        dict(
-            type='PerspectiveCameras',
-            convention='opencv',
-            in_ndc=False,
-            focal_length=focal_length,
-            image_size=image_size,
-            principal_point=camera_center))
+    def __init__(self, *args, **kwargs):
+        super(GTAHumanConverter, self).__init__(*args, **kwargs)
+
```

Comment:
Do we need to specify where this name comes from as a comment?

Reply:
Fixed in https://github.com/open-mmlab/mmhuman3d/pull/87/commits/6bcb5fa26c02b5fed031865afa9c6929106d6ac7.